astroquery/jplhorizons/tests/test_jplhorizons_remote.py

```diff
@@ -405,3 +409,51 @@ def test_ephemerides_extraprecision(self):
         vec_highprec = obj.ephemerides(extra_precision=True)
 
         assert (vec_simple['RA'][0]-vec_highprec['RA'][0]) > 1e-7
+
+    def test_geodetic_queries(self):
+        """
+        black-box test for observer and vectors queries with geodetic
+        coordinates. checks spatial sensibility.
+        """
+        phobos = {
+            'body': 401,
+            'lon': random.randint(-150, -1),
+            'lat': random.randint(-80, 80),
```

Comment:
I'm not sure running the tests on random inputs is worth the effort. Even if the tests stumble upon a weird edge case that causes the tests to fail, it can be very difficult to figure out what the problematic values were, which prevents them from being addressed.

Reply:
It makes me nervous to use only white-box tests, but I understand if you prefer it for stability reasons. So I will turn this into another explicit equality test.

Reply:
The best way of doing randomized tests would be to use [`hypothesis`](https://hypothesis.readthedocs.io/en/latest/) because it is capable of storing inputs that caused a failure, but `astroquery` isn't using it at the moment.

Reply:
I do like `hypothesis`. But this is totally fine with me. Weird edge cases are not actually that likely here.